dali/server/daldap.cpp

```diff
@@ -164,19 +164,23 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
 
                 CDateTime now;
                 now.setNow();
-                if (now.compare(reqUTCTimestamp) < 0)//timestamp from the future?
+                if (now.compare(reqUTCTimestamp, false) < 0)//timestamp from the future?
```

Comment:
How do we ensure the req timestamp is  in UTC?

Reply:
it is set by the caller (runtime engine), in dasess.cpp method createDaliSignature, which does this
        StringBuffer timeStr;
        now.setNow();
        now.getString(timeStr, false);//get UTC timestamp